src/main/java/com/google/step/snippet/servlets/SearchServlet.java

```diff
@@ -43,9 +49,16 @@
   private static final String STACK_CSE_ID = "INSERT_STACKOVERFLOW_CSE_ID";
   private static final String GEEKS_CSE_ID = "INSERT_GEEKSFORGEEKS_CSE_ID";
   private static final String API_KEY = "INSERT_API_KEY";
-  private static final String CSE_URL = "https://www.googleapis.com/customsearch/v1";
   private static final String CSE_ITEMS = "items";
   private static final String CSE_LINK = "link";
+  private static final String CSE_URL = "https://www.googleapis.com/customsearch/v1";
+
+  private final List<Client> clients =
+      new ArrayList<>(
```

Comment:
I think you can just do `Arrays.asList` without explicitly creating a new ArrayList? `Arrays.asList` already returns a List. It compiles for me.